README.md

```diff
@@ -25,14 +25,8 @@ To request some content for a Meetup you will be hosting, or to help clear up an
 - This would help me...
   * An explanation of how this content would help you
 - Resources for getting started...
-  * Any current documentation or pages that exist on the topic. Check _at least_ these spots:
-    * [Developer Getting Started](https://blockstack.org/devs)
-    * [Blockstack Forum](https://forum.blockstack.org/)
-    * [Tutorials](https://blockstack.org/tutorials)
-    * [Blockstack Core Tech Docs](https://github.com/blockstack/blockstack-core/tree/master/docs)
-    * [Blockstack Browser Tech Docs](https://github.com/blockstack/blockstack-browser/tree/master/docs)
-
-
+  * Any current documentation or pages that exist on the topic. Search _at least_ this spots:
```

Comment:
typo should be "this spot" or you could do "this documentation" . Thanks for this PR @friedger

Reply:
Fixed